metrics/resource_view.go

```diff
@@ -94,7 +111,9 @@ func UnregisterResourceView(views ...*view.View) {
 	resourceViews.lock.Lock()
 	defer resourceViews.lock.Unlock()
 
-	for _, meter := range allMeters.meters {
+	for _, key := range allMeters.meters.Keys() {
```

Comment:
It's unfortunate that we may need to generate and throw away a slice here... it would be nice if there was an iteration interface on the Cache.

Still, this is run fairly infrequently (only when registering views, which should be at startup time).